src/main/java/com/denizenscript/depenizen/bukkit/objects/towny/NationTag.java

```diff
@@ -224,6 +226,23 @@ public static void registerTags() {
             return list;
         });
 
+        // <--[tag]
+        // @attribute <NationTag.get_rank[(<rank>)]>
+        // @returns ListTag(PlayerTag)
+        // @plugin Depenizen, Towny
+        // @description
+        // Returns a list of the nation's members with a given rank.
+        // -->
+        tagProcessor.registerTag(ListTag.class, "get_rank", (attribute, object) -> {
+            String rank = attribute.getParam();
```

Comment:
Should use the register with-param method rather than reading the param inside the method body